src/services/users/fetch-users.test.js

```diff
@@ -0,0 +1,27 @@
+const { expect } = require('chai')
+const {fixtures , setupDB} = require('../../../test/utils')
+const fetchUsers = require('./fetch-users')
+
+describe('Fetch all users', () => {
+	const numberOfUsers = 5
+	let users
+	beforeEach(async () => {
+		await setupDB()
+		users = await fixtures.insertMultipleUsers(numberOfUsers)
+	 })
+
+	describe('Success', () => {
+
+		it('should return the right response', async () =>{
+			const actualUsers = await fetchUsers()
+			return expect(actualUsers.length).to.eql(users.rowCount)	
```

Comment:
So `right response` is about the actual data being returned by the function being correct not about the number fetched being correct or wrong

Reply:
yeah, i get you. thank you